docs/how-to/build-a-12-factor-app-rock.md

```diff
@@ -0,0 +1,157 @@
+# How to build a 12-Factor app rock
+
```

Comment:
looking at the [rendered page](https://canonical-ubuntu-documentation-library--735.com.readthedocs.build/rockcraft/en/735/how-to/build-a-12-factor-app-rock/#include-extra-files-in-the-oci-image) I felt like the transition from 'how to build a 12-factor app rock' to 'include extra files in the OCI image' was quite abrupt; should we get some kind of preamble here pointing out that these sections are how-tos for 12-factor rocks?

Reply:
CC @jdkandersson 

Here's a one-sentence attempt at a preamble; comments and feedback are appreciated. Since the contents are listed on the right-hand side, I opted to not repeat the contents again.

> The following how-to guides provide instructions on managing and configuring rocks for 12-factor applications.

Reply:
Ok let's go with it

Reply:
Thanks, I added the preamble